### DISCUSSION TITLE
How to add to PR some changed files during renovate update ?

### DISCUSSION THREAD

**Discussion by MathRobin** [renovatebot/renovate#26547] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I have some private npm packages who can change some project files. When i upgrade version of my package, renovate see it, creates the pull request, run the build/test workflow correctly, if workflow is good, automerge works.

But as said, some of them can update my current projet. For example, here, i've a private package which can update `.editorconfig` file (not only but was easier with this one).

Here the diff in the PR:
![2024-01-08_12-01](https://github.com/renovatebot/renovate/assets/1046585/c7e2f8dc-96f9-43d8-a8a3-ffacff57f6c7)
Only package.json and yarn.lock are updated

But in my workflow if I do a `git status`, i can see this:
![2024-01-08_12-02](https://github.com/renovatebot/renovate/assets/1046585/b92da39f-54db-43a3-a10f-13625f1f41a1)

I would like to see `.editorconfig` file appear in my PR. Without this, i have to pull my repo on local, redo the full build and commit it again. This private package (our `api-build`) is a dependency of more than twenty repo in our organisation, manage a lot of things and generates/updates more than thirty files in each project. Update manually each project after each build is a real pain (and a big waste of time). We would really appreciate your help :smiley: 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by viceice** [comment]
scripts are disabled on hosted GitHub app for security reasons. you need to self-Host when you needs scripts to be executed.
